eve/main.yml

```
@@ -0,0 +1,31 @@
+---
+version: '0.2'
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker:
+      type: local
+    steps:
+    - TriggerStages:
+        stage_names:
+        - gradle-build
+  gradle-build:
+    worker:
+      type: kube_pod
+      path: eve/workers/pod.yaml
+      images:
+        build: eve/workers/build
+    steps:
+      - Git: &git_pull
+          name: git pull
+          repourl: "%(prop:git_reference)s"
+          shallow: true
+          retryFetch: true
+          haltOnFailure: true
+      - ShellCommand:
+          name: gradle build
+          command: gradle build
```

Let's add a new line here
```suggestion
          command: gradle build

```